databrowser/src/pages/datasets/overview/OverviewListPage.vue

```diff
@@ -366,12 +383,50 @@ const hideFilters = () => {
   isFiltersModalVisible.value = false;
 };
 
+const filterSelectedForComponent = ref<{ key: string; value: string }[]>([]);
+
+const initializeFiltersAndSearch = () => {
+  const {rawfilter, searchQuery} = getStartedQuery();
+
+  if (!searchQuery && !rawfilter) {
```

Comment:
Watch out for implicit type casts, are you sure that any developer that touches this code knows whats happening? I bet that's not the case ;)

So in my opinion it's better to do explicit comparison, e.g. `searchQuery.length === 0 && rawfilter.length === 0` or similar